### PR TITLE
Restrict the temperature update using provided parameters

### PR DESCRIPTION
**Pull request by totto82**
We may want to change the default values, but for now they seem good enough.

### CODE REVIEW

**Comment by totto82** [comment]
jenkins build this please

**Review by bska — COMMENTED** [review]
Looks good for the most part.  I have one question and one suggestion.  Otherwise I'm happy with this change.

**Comment by totto82** [comment]
jenkins build this please

File: opm/models/blackoil/blackoilnewtonmethod.hh [hunk 1]
```diff
@@ -334,9 +367,9 @@ protected:
             if (enableBrine && pvIdx == Indices::saltConcentrationIdx)
                 nextValue[pvIdx] = std::max(nextValue[pvIdx], 0.0);
 
-            // keep the temperature above 100 and below 1000 Kelvin
+            // keep the temperature within given values
             if (enableEnergy && pvIdx == Indices::temperatureIdx)
-                nextValue[pvIdx] = std::max(std::min(nextValue[pvIdx], 1000.0), 100.0);
+                nextValue[pvIdx] = std::max(std::min(nextValue[pvIdx], tempMax_), tempMin_);
```
[the remark this answers is not in the document]
Comment (totto82) [reply opm/models/blackoil/blackoilnewtonmethod.hh]: I will start using clamp. I always mix min and max when I manually try to clamp values. Thanks for the tip

**Review by bska — APPROVED** [review]
Thanks for the updates.  I'll merge this into master now.

**Pull request merged**